packages/happo-target-firefox/src/waitForImagesToRender.js

```diff
@@ -1,7 +1,13 @@
 import findBackgroundImageUrls from './findBackgroundImageUrls';
 
-function waitForImageToLoad(url) {
+export function waitForImageToLoad(url) {
   return new Promise((resolve, reject) => {
+    // No need to wait for the image to be loaded.
+    if (url.indexOf('data:image') === 0) {
```

Comment:
I have been digging a bit more in the issue. It seems to be a char encoding issue. I haven't went down the rabbit hole yet. Are you saying that even with not network latency, it's preferable to load the image first?